modules/Bio/EnsEMBL/Analysis/Hive/Config/production_rnaseq_db.pm

```diff
@@ -0,0 +1,663 @@
+    'registry_host'             => '', # host for registry db
+    'registry_port'             => '', # port for registry db
+    'registry_db'               => '', # name for registry db
+
+    'rnaseq_summary_file'       => '' || catfile($self->o('rnaseq_dir'), $self->o('species_name').'.csv'), # Set this if you have a pre-existing cvs file with the expected columns
+    'rnaseq_summary_file_genus' => '' || catfile($self->o('rnaseq_dir'), $self->o('species_name').'_gen.csv'), # Set this if you have a pre-existing genus level cvs file with the expected columns
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'species_name'              => '', # e.g. mus_musculus
+    'production_name'           => '', # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'taxon_id'                  => '', # should be in the assembly report file
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'assembly_name'             => '', # Name (as it appears in the assembly report file)
+    'assembly_accession'        => '', # Versioned GCA assembly accession, e.g. GCA_001857705.1
+
+    'skip_rnaseq'               => '0', # Will skip rnaseq analyses if 1
+
+    'mapping_required'          => '0', # If set to 1 this will run stable_id mapping sometime in the future. At the moment it does nothing
+    'mapping_db'                => '', # Tied to mapping_required being set to 1, we should have a mapping db defined in this case, leave undef for now
+
+    'uniprot_version'           => 'uniprot_2019_04', # What UniProt data dir to use for various analyses
+
+    # Keys for custom loading, only set/modify if that's what you're doing
+    'protein_entry_loc'            => catfile($self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'entry_loc'), # Used by genscan blasts and optimise daf/paf. Don't change unless you know what you're doing
+
+
+########################
+# Pipe and ref db info
+########################
+    'rnaseq_db_server'             => $self->o('databases_server'),
+    'rnaseq_db_port'               => $self->o('databases_port'),
+
+    'rnaseq_refine_db_server'       => $self->o('databases_server'),
+    'rnaseq_refine_db_port'         => $self->o('databases_port'),
+
+    'rnaseq_blast_db_server'       => $self->o('databases_server'),
+    'rnaseq_blast_db_port'         => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'         => 'mysql-ens-meta-prod-1',
+    'production_db_port'           => '4483',
+
+
+    databases_to_delete => ['rnaseq_blast_db', 'rnaseq_refine_db', 'rnaseq_db'],
```

Comment:
I should have explained what is the aim of the databases_to_delete. The idea is there is a flag in HiveBaseConfig, drop_databases, which should only be used on the command line. If it is set to 1, it will delete the databases in this array. It is to be able to delete the databases created during this pipeline if you want to restart it using the same setting. In this array there should only be databases created in the pipeline, usually with a 'create_*_db' using 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreateDatabase'
This array should only have rnaseq_db. rnaseq_blast_db and rnaseq_refine_db are created by the "old" RNA-seq pipeline and we definitely want to keep them if they exist.